ui/lib/core/addon/components/form-field.hbs

```diff
@@ -16,13 +16,13 @@
             <G.Legend data-test-form-field-label>{{this.labelString}}</G.Legend>
           {{/if}}
           {{#if this.helpTextString}}
-            <G.HelperText data-test-help-text>{{this.helpTextString}}</G.HelperText>
+            <G.HelperText data-test-form-field-help-text>{{this.helpTextString}}</G.HelperText>
           {{/if}}
           {{#if @attr.options.subText}}
-            <G.HelperText data-test-label-subtext>
+            <G.HelperText data-test-form-field-subtext>
               {{@attr.options.subText}}
               {{#if @attr.options.docLink}}
-                <DocLink @path={{@attr.options.docLink}}>See our documentation</DocLink>
+                <DocLink @path={{@attr.options.docLink}} data-test-form-field-doc-link>See our documentation</DocLink>
```

Comment:
Similar to above, add this to the FORMs area of the GENERAL selectors and pass in the link so we can personalize it while doing the tests.
```suggestion
                <DocLink @path={{@attr.options.docLink}} data-test-doc-link={{@attr.options.dockLink}}>See our documentation</DocLink>
```